pipelines/fbc-release/fbc-release.yaml

```diff
@@ -221,9 +221,9 @@ spec:
         resolver: "git"
         params:
           - name: url
-            value: https://github.com/redhat-appstudio/release-service-catalog.git
+            value: https://github.com/theflockers/release-service-catalog.git
           - name: revision
-            value: main
+            value: rhtaprel_382
```

Comment:
should be changed before merge.

Reply:
Let's keep this unresolved then so it lowers the chance of it accidentally being merged with it like this

Reply:
Indeed. I did resolve it as I had actually removed this, but put it back as the code was changed and I had to re-test.